components/scream/cime_config/namelist_defaults_scream.xml

```diff
@@ -189,6 +189,12 @@ tweaking their $case/namelist_scream.xml file.
     <!-- Radiation -->
     <rrtmgp inherit="physics_proc_base">
       <active_gases>h2o, co2, o3, n2o, co, ch4, o2, n2</active_gases>
+      <ch4vmr>-1.0</ch4vmr>
```

Comment:
It looks like this is setting the default values for these gases. I think we want to pull these from the F2010 use case from EAM and set here.